moveit_ros/planning_interface/move_group_interface/include/moveit/move_group_interface/move_group_interface.h

```diff
@@ -756,9 +756,16 @@ class MOVEIT_MOVE_GROUP_INTERFACE_EXPORT MoveGroupInterface
       Return a value that is between 0.0 and 1.0 indicating the fraction of the path achieved as described by the
      waypoints.
       Return -1.0 in case of error. */
+  [[deprecated("Drop jump_threshold")]] double  //
```

Comment:
Replace with your better worded deprecation message from before:

```
[[deprecated("Replace JumpThreshold with CartesianPrecision")]]
```

Reply:
There is no CartesianPrecision argument needed. Hence, a different message.